stubs/pexpect/pexpect/pty_spawn.pyi

```diff
@@ -0,0 +1,74 @@
+from _typeshed import Incomplete
+from collections.abc import Callable, Sequence
+from io import TextIOWrapper
+from os import _Environ
+from typing import AnyStr
```

Comment:
`AnyStr` is not an alias for `str | bytes`, which is most likely what you want to use instead below.